### ISSUE TITLE
Some emoji are broken 

### ISSUE THREAD

**Issue by estellecomment** [element-hq/element-web#26496]
### Steps to reproduce

1. Where are you starting? What can you see?
Add a reaction to a message

#### What did you expect?
All emojis are available in menu. Once added, emoji is displayed.

#### What happened instead?
Some emoji are not displayed, they have a broken image. They can be added, but the broken image is displayed.
They are:
shaking_face
pink_heart light_blue_heart grey_heart
leftwards/rightwards_pushing_hand
moose donkey wing goose jellyfish
hyacinth
ginger_root pea_pod
folding_hand_fan hair_pick
wireless

<img width="338" alt="Screen Shot 2023-11-02 at 5 41 49 PM" src="https://github.com/vector-im/element-web/assets/911434/61bd698d-504e-440d-a0b6-88e9e1133068">

<img width="314" alt="image" src="https://github.com/vector-im/element-web/assets/911434/82a7dcdf-4a25-4a2b-9985-e2c322841f74">

### Operating system

macOS Monterey (12.6)

### Browser information

Firefox 119.0 (64 bits)

### URL for webapp

https://app.element.io

### Application version

Element : 1.11.47, Olm : 3.2.14

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by t3chguy** [comment]
This is due to us having to rollback to SBIX fonts and Safari regressing their COLR support - https://bugs.webkit.org/show_bug.cgi?id=262223

**Comment by Destroyer** [comment]
I have the same issue on windows, very annoying

![image](https://github.com/element-hq/element-web/assets/289988/70187532-0fc6-4c2a-824d-e986dafae327)

**Comment by karenmcd** [comment]
![Screenshot from 2024-06-29 23-03-40](https://github.com/element-hq/element-web/assets/37629978/40282de5-e32f-4a27-9033-36ad30120056)


Same, 
```
inxi -Fz                                                                                                               ✔ 
System:
  Kernel: 6.9.3-3-MANJARO arch: x86_64 bits: 64
  Desktop: GNOME v: 46.2 Distro: Manjaro Linux
Machine:
  Type: Desktop Mobo: ASUSTeK model: PRIME X570-P v: Rev X.0x
    serial: <superuser required> BIOS: American Megatrends v: 5013
    date: 03/22/2024
CPU:
  Info: 8-core model: AMD Ryzen 7 5800X bits: 64 type: MT MCP cache: L2: 4 MiB
  Speed (MHz): avg: 3839 min/max: 2200/4850 cores: 1: 3964 2: 4759 3: 3802
    4: 4627 5: 3802 6: 3804 7: 3802 8: 3832 9: 3821 10: 4741 11: 3799 12: 3832
    13: 3802 14: 2200 15: 3045 16: 3805
Graphics:
  Device-1: NVIDIA GA104 [GeForce RTX 3060 Ti Lite Hash Rate] driver: nvidia
    v: 550.90.07
  Display: x11 server: X.org v: 1.21.1.13 with: Xwayland v: 24.1.0 driver:
    X: loaded: N/A failed: nvidia gpu: nvidia,nvidia-nvswitch note: X driver
    n/a, try sudo/root resolution: 1920x1080~60Hz
  API: EGL v: 1.5 drivers: kms_swrast,nvidia,swrast
    platforms: gbm,x11,surfaceless,device
  API: OpenGL v: 4.6.0 compat-v: 4.5 vendor: nvidia mesa v: 550.90.07
    renderer: NVIDIA GeForce RTX 3060 Ti/PCIe/SSE2
Audio:
  Device-1: NVIDIA GA104 High Definition Audio driver: snd_hda_intel
  Device-2: AMD Starship/Matisse HD Audio driver: snd_hda_intel
  Device-3: Jieli UACDemoV1.0 driver: hid-generic,snd-usb-audio,usbhid
    type: USB
  API: ALSA v: k6.9.3-3-MANJARO status: kernel-api
  Server-1: PipeWire v: 1.0.7 status: active
Network:
  Device-1: Realtek RTL8111/8168/8211/8411 PCI Express Gigabit Ethernet
    driver: r8169
  IF: enp4s0 state: up speed: 1000 Mbps duplex: full mac: <filter>
Drives:
  Local Storage: total: 1.78 TiB used: 879.38 GiB (48.2%)
  ID-1: /dev/nvme0n1 vendor: Crucial model: CT1000P1SSD8 size: 931.51 GiB
  ID-2: /dev/sda vendor: Crucial model: CT960BX500SSD1 size: 894.25 GiB
Partition:
  ID-1: / size: 109.44 GiB used: 11.36 GiB (10.4%) fs: ext4
    dev: /dev/nvme0n1p1
  ID-2: /home size: 776.48 GiB used: 149.51 GiB (19.3%) fs: ext4
    dev: /dev/nvme0n1p3
Swap:
  ID-1: swap-1 type: file size: 512 MiB used: 511.8 MiB (100.0%)
    file: /swapfile
Sensors:
  System Temperatures: cpu: 57.9 C mobo: N/A gpu: nvidia temp: 40 C
  Fan Speeds (rpm): N/A gpu: nvidia fan: 0%
Info:
  Memory: total: 32 GiB available: 31.26 GiB used: 12.55 GiB (40.2%)
  Processes: 392 Uptime: 3d 3m Shell: Zsh inxi: 3.3.35
```

**Comment by karenmcd** [comment]
On element-desktop v 1.11.65-1 I clicked on my Avatar,
chose "all settings", then "appearance", then "show advanced".
In there I unchecked "use bundled emjoi font" and then closed the menu to see it still was broken. Yes.
Then I went back to that advanced appearance menu and put a check back in "use bundled emoji font" and also added a check into "use a more compact and 'modern' layout", and closed the menu and now the emoji works as expected.
![Screenshot from 2024-06-30 21-16-10](https://github.com/element-hq/element-web/assets/37629978/1128ab38-d556-45b8-94db-2af4d51fdce1)
